endpoints/openrtb2/auction.go

```diff
@@ -567,7 +567,7 @@ func (deps *endpointDeps) validateImp(imp *openrtb2.Imp, aliases map[string]stri
 	return nil
 }
 
-func validateBanner(banner *openrtb2.Banner, impIndex int) error {
+func validateBanner(banner *openrtb2.Banner, imp *openrtb2.Imp, impIndex int) error {
```

Comment:
I'd suggest passing in a boolean flag like `isInterstitial` that is set if `imp.Instl == 1` rather than passing the entire `imp`

Reply:
I tried to work on this and understand the idea but not exactly sure how to go about accessing the imp.Instl property without the imp object to set the bool flag. Would I be setting the flag outside this function using the imp object there or am I off base?

Reply:
Yup. You're right. You'd have to set the boolean flag outside this function and then pass it to this function when it is called.
You can do this by changing the signature of this function to be:
```
func validateBanner(banner *openrtb2.Banner, impIndex int, isInterstitial bool) 
```

Reply:
@mansinahar I know there probably is a better place for the interstitial function but wanted to make sure I was on the right track?